examples/java/hello_world/MainActivity.java

```diff
@@ -26,7 +26,7 @@
 
 public class MainActivity extends Activity {
   private static final String ENDPOINT =
-      "http://0.0.0.0:9001/api.lyft.com/static/demo/hello_world.txt";
+      "http://s3.amazonaws.com/api.lyft.com/static/demo/hello_world.txt";
```

Comment:
Envoy won't be proxying these requests if they aren't pointed at the address/port it's listening to. Maybe merge master into this branch to pull in the config fixes that merged yesterday?

Reply:
It didn't seem to work but we can try again